.github/workflows/prod-deploy.yaml

```diff
@@ -5,13 +5,37 @@ on:
     types: [published]
 
 jobs:
+  create-release:
+    name: 'Create release tar'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v3.0.0
+      - name: Build
+        run: |
+          mkdir dist
+          cd dist
+          npm ci
```

Comment:
It looks like you are just making `dist` from node modules. To get dist folder we first need
* `npm ci`
* `npm run build`
 
Then there will be a dist folder which then can be used to make tarball.

Also I don't get why we are doing this duplicate build because on line 44, `jsmrcaga/action-netlify-deploy@v1.7.2` will build it again for netlify. So why not use that for both builds?

Reply:
> It looks like you are just making dist from node modules. To get dist folder we first need

Thanks, I obviously know very, very little about how npm works :)

> Also I don't get why we are doing this duplicate build because on line 44, jsmrcaga/action-netlify-deploy@v1.7.2 will build it again for netlify. So why not use dist from that for tarball too?

Because the point is to avoid having third party repos in the TCB for the release tar, relying on `jsmrcaga` would keep him in the TCB, which would defeat the point here.

Reply:
But it's still in TCB by building for app.cinny.in, IMHO all we are doing by this is increasing build times by doing the same thing twice.

Reply:
> But it's still in TCB by building for app.cinny.in

Yep, I'm scratching my own itch a bit here - I don't use app.cinny.in, so I'm admittedly a bit less concerned there. I'd suggest very strongly you look at using https://github.com/netlify/actions instead, but trying to make that change here would be quite a bit of work since I don't have a netlify account or any sites with which I could test this. You're welcome to take or leave my contribution here.

> IMHO all we are doing by this is increasing build times by doing the same thing twice.

It gets run in parallel, so it doesn't slow anything down (and also takes a few seconds in total anyway...)

Reply:
The good news about doing the build separately from the upload is it doesn't really matter how maintained the netlify actions script is, you just need to use it to upload a static directory, see https://github.com/netlify/actions/tree/master/cli. If you're really worried you could use their CLI tool directly, but I'm not sure its worth it.

Reply:
Actually, I take that back, the issues on netlify/actions where things didn't build correctly....turns out its because of *its* dependencies being maliciously overwritten by authors...guess it does make sense to reduce the TCB of actions runs :p. It should be incredibly trivial to just install their cli tool and use that - https://github.com/netlify/cli